crates/net/network/src/frost/manager.rs

```
@@ -215,6 +215,29 @@ impl FrostManager {
         self.authorities.contains_key(peer_id)
     }
 
+    fn respond_with_status(
+        sender: tokio::sync::oneshot::Sender<ConnectionEstablishedStatus>,
+        status: ConnectionEstablishedStatus,
+        message: &str,
+        peer_id: &PeerId,
+    ) -> bool {
+        if sender.is_closed() {
+            return false;
+        }
+        let msg = format!(
+            "Received FrostProtocolEvent::ConnectionEstablished event from peer {:?}. Error: {:?}",
+            peer_id, message
+        );
+        if sender.send(status).is_err() {
+            warn!(
```

Nitpick: Why not just be quiet here and return `false` - removing that `warn!`? You can probably remove this method entirely and respond directly, ignoring any errors.

---

I just kept it for informational purposes. If the sending errors, at least we would know that. If you insist, I can remove it